src/api/customer/customer-validator.ts

```diff
@@ -1,39 +1,49 @@
-import { celebrate, Joi, Segments } from 'celebrate';
+import { celebrate, Joi, Segments } from "celebrate";
 
 const createCustomerValidator = () => {
-    return celebrate({
-        [Segments.BODY]: Joi.object().keys({
-            name: Joi
-                .string()
-                .trim()
-                .min(2)
-                .max(100)
-                .required(),
-            document: Joi.object().keys({
-                cpf: Joi
-                    .string()
-                    .length(11)
-                    .regex(/^\d+$/),
-                cnpj: Joi
-                    .string()
-                    .length(14)
-                    .regex(/^\d+$/)
-            }).required().xor("cpf", "cnpj"),
-            contact: Joi.object().keys({
-                email: Joi
-                    .string()
-                    .email()
-                    .max(100)
-                    .required(),
-                phone: Joi
-                    .string()
-                    .trim()
-                    .max(15)
-            }).required()
+  return celebrate({
+    [Segments.BODY]: Joi.object().keys({
+      name: Joi.string().trim().min(2).max(100).required(),
+      document: Joi.object()
+        .keys({
+          cpf: Joi.string().length(11).regex(/^\d+$/),
+          cnpj: Joi.string().length(14).regex(/^\d+$/),
         })
-    });
+        .required()
+        .xor("cpf", "cnpj"),
+      contact: Joi.object()
+        .keys({
+          email: Joi.string().email().max(100).required(),
+          phone: Joi.string().trim().max(15),
+        })
+        .required(),
+    }),
+  });
+};
+
+const listCustomersValidator = () => {
+  return celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      offset: Joi.number().optional().min(0),
```

Comment:
Poderia ter especificado que esse valor deveria ser um número inteiro, para não aceitar frações (ex `1.52`)

```suggestion
      offset: Joi.number().integer().optional().min(0),
```